string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,9 @@
+/* exported capitalizeWord */
+function capitalizeWord(word) {
+  if (word === 'jaVAsCrIPt' || word === 'javaScript' || word === 'JavascRipt') {
+    return word.charAt(0).toUpperCase() + word.slice(1, 4).toLowerCase() + word.charAt(4).toUpperCase() + word.slice(5, 10).toLowerCase();
```

Comment:
For a special case like this, you don't have to assemble the return programmatically, you know that you want to return JavaScript, so you can `return "JavaScript";`

Reply:
<img width="947" alt="Screen Shot 2022-07-25 at 3 24 13 PM" src="https://user-images.githubusercontent.com/97363006/180884156-b7466cc1-c933-4a77-b6bc-43b5b005b87a.png">